packages/server/graphql/queries/helpers/fetchAllRepoIntegrations.ts

```diff
@@ -13,13 +14,25 @@ const fetchAllRepoIntegrations = async (
   const {dataLoader} = context
 
   const permLookup = await getPermsByTaskService(dataLoader, teamId, userId)
-  const [prevRepoIntegrations, jiraProjects, githubRepos, jiraServerProjects] = await Promise.all([
+  const [
+    prevRepoIntegrations,
+    jiraProjects,
+    githubRepos,
+    gitlabProjects,
+    jiraServerProjects
+  ] = await Promise.all([
     getPrevRepoIntegrations(userId, teamId, permLookup),
     dataLoader.get('allJiraProjects').load({teamId, userId}),
     fetchGitHubRepos(teamId, userId, dataLoader, context, info),
+    fetchGitLabProjects(teamId, userId, dataLoader, context, info),
```

Comment:
This file is being used in `allAvailableRepoIntegrations` and `repoIntegrations` in the `TeamMember` object. 

As mentioned [here](https://github.com/ParabolInc/parabol/pull/6267/files#r835236268), we're not using `repoIntegrations` in the Poker GitLab integrations, unlike GitHub or Jira. 

However, initially, I started following the `repoIntegration` pattern and created the `fetchGitLabProjects` file. I think it's best to keep it because we'll need it in the task integration, where we do want to display all `repoIntegrations`. There's an issue for this here: https://github.com/ParabolInc/parabol/issues/6050

I haven't updated `getPrevRepoIntegrations` to account for GitLab because we don't need it in Poker, and it'll be refactored anyway - see issue: https://github.com/ParabolInc/parabol/issues/6007